### PR TITLE
Rfink/do not use ws rs family

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

**Comment by uschi2000** [comment]
fixes #594

**Comment by justinuang** [comment]
I think it's in several other places as well.

screen shot 2017-11-03 at 6 17 07 pm

**Comment by uschi2000** [comment]
done

**Comment by uschi2000** [comment]
all the other libraries on the classpath can obviously still run this code path, so it's not clear to me that fixing it in http-remoting is going to fully resolve your problem.

File: okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/metrics/HostMetrics.java [hunk 1]
```diff
@@ -40,11 +40,11 @@
     /** Creates a metrics registry for calls from the given service to the given host. */
     public HostMetrics(TaggedMetricRegistry registry, String serviceName, String hostname) {
         informational = registry.meter(name(serviceName, hostname, "informational"));
-        successful    = registry.meter(name(serviceName, hostname, "successful"));
-        redirection   = registry.meter(name(serviceName, hostname, "redirection"));
-        clientError   = registry.meter(name(serviceName, hostname, "client-error"));
-        serverError   = registry.meter(name(serviceName, hostname, "server-error"));
-        other         = registry.meter(name(serviceName, hostname, "other"));
```
Comment (ellisjoe) [inline okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/metrics/HostMetrics.java:47]: lame

**Review by ellisjoe — COMMENTED** [review]
sad. what did they break exactly?

**Comment by uschi2000** [comment]
unclear who broke what, but in Hadoop-reality there are conflicting versions of this class.

**Comment by uschi2000** [comment]
sort of a miracle that none of the other libraries would hit this, but the sacrifice in http-remoting is small if it helps.

**Comment by uschi2000** [comment]
+1, @ellisjoe ?

**Review by ellisjoe — APPROVED** [review]

**Pull request merged**